### ISSUE TITLE
Bug: indented text causing problems in iPad slide over mode

### ISSUE THREAD

**Issue by TUSAB-web** [Dimillian/IceCubesApp#1798]
## Environment:

 - OS: 
 iPadOS 17.1.2
 - IceCubesApp version: 
 1.10.5

## Description

A recent update has indented the text of all toots. It is causing issues, including messed up display of boost and favorites icons and counts…

## Related Issues

 - [ ] Search that this bugs don't already exist before creating it.
<img width="376" alt="httpsgithub comDimillianIceCubesAppissuesnewassignees= labels=bug projects= template=bug md title=Bug3A jpeg" src="https://github.com/Dimillian/IceCubesApp/assets/52001038/b0748df5-c85d-4b45-a165-7babd399dd8e">

**Comment by TUSAB-web** [comment]
Another image, this time to show one that hasn’t been further indented as part of a thread. Notice how the number of boosts displays as 2 over 2 instead of 22.

<img width="357" alt="Incorrect boost number display jpeg" src="https://github.com/Dimillian/IceCubesApp/assets/52001038/ef0ccc19-d80d-4b9e-8276-37c2957239c8">

**Comment by Dimillian** [comment]
Well I guess if you use a view that narrow I'll disable the count in this case.

**Comment by TUSAB-web** [comment]
I didn’t see the problem in prior versions. Don’t have an old version here, but is it possible the bar of icons wasn’t indented with the text before? As in, the bar of icons started farther to the left? Or maybe it’s the … wasn’t there before? Sorry can’t recall exactly how it looked before the update

**Comment by Dimillian** [comment]
Yes before it was not indented. But if you're using the avatar .top layout like in the display settings like before it'll be fine. 
I'm wondering how to fix it for small display now.

**Comment by TUSAB-web** [comment]
Ahh… thank you! I just switched the avatar position back to top on my iPad, which fixes it for me for now.

**Comment by Dimillian** [comment]
I still want to find a fix for the slideover mode, I guess I'll switch to some compact layout

**Comment by TUSAB-web** [comment]
> I still want to find a fix for the slideover mode, I guess I'll switch to some compact layout

As a web developer, I’d just move the boost/favorite icon bar into a div outside of the div containing the indented text… not sure how that applies to app development though.

**Comment by Dimillian** [comment]
Here we have size class, I'll just remove the count if the size class is compact + is on macOS / iPadOS.

**Comment by Dimillian** [comment]
Here we go, but I'm planing more fix for slidover, including showing the tab bar instead.
